Makefile

```diff
@@ -51,18 +51,22 @@ _output/share/lima/lima-guestagent.Linux-aarch64:
 	chmod 644 $@
 
 .PHONY: install
+install: DEST := $(realpath $(DESTDIR)/$(PREFIX))
 install:
-	cp -av _output/* "$(DESTDIR)/$(PREFIX)/"
-	if [[ $(shell uname -s ) != Linux && ! -e "$(DESTDIR)/$(PREFIX)/bin/nerdctl" ]]; then ln -sf nerdctl.lima "$(DESTDIR)/$(PREFIX)/bin/nerdctl"; fi
+	@test "$(DEST)" || (echo "invalid destination: $(DESTDIR)$(PREFIX)"; exit 1)
```

Comment:
Can we mkdir -p when DEST does not exist?

Reply:
We could use `$(abspath ...)`. Lemme send another commit with it, 1 min :)

Reply:
Please take a look at the following commit :)